components/alibi-explain-server/pyproject.toml

```diff
@@ -6,19 +6,21 @@ authors = ["Seldon Technologies Ltd. <hello@seldon.io>"]
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
-python = "^3.7"
-alibi = "0.6.2"
+python = "^3.7,<3.8"
+alibi = "0.6.4"
 numpy = "1.19.5"
-tensorflow = "2.5.1"
-scikit-learn = "0.24.2"
-lightgbm = "3.1.1"
-xgboost = "1.4.2"
-catboost = "0.24.4"
+numba = "0.55.0"
+tensorflow = "2.7.0"
```

Comment:
In case of numpy and tensorflow I kind of wanted to have more explicit control on what version will end up in the build

Reply:
So the reason for which tensorflow is more explicit is the same as why other one are (xgboost, scikit-learn, etc) -> so that it is easier at glance to see what versions of these are meant to be installed in the server.

Conscious decision vs effect of dependency resolution.